examples/huggingface/neural_speed/run_accuracy.py

```diff
@@ -25,22 +25,22 @@
     args = parser.parse_args()
     print(args)
     model_args=f'pretrained="{args.model_name}",dtype=float32,trust_remote_code=True'
-    if args.use_gptq:
-        model_args += ",use_gptq=True"
```

Comment:
seems one args removed here~ 
no problem right ? @zhentaoyu

Reply:
no effect on `NS` since it will be handled by model_config.